solana/programs/matching-engine/src/processor/auction/execute_fast_order/mod.rs

```diff
@@ -240,3 +95,210 @@ fn handle_execute_fast_order<'info>(
         order_executed_event,
     })
 }
+
+pub struct ActiveAuctionAccountInfos<'ix> {
+    pub auction_best_offer_token: AccountInfo<'ix>,
+    pub auction_executor_token: AccountInfo<'ix>,
+    pub auction_initial_offer_token: AccountInfo<'ix>,
+    pub auction_custody_token: AccountInfo<'ix>,
+    pub active_auction: AccountInfo<'ix>,
+    pub auction_custodian: AccountInfo<'ix>,
+}
```

Comment:
We should try to use refs of AccountInfo instead of having to clone them.

Reply:
After reading through the helper method below again, it might make more sense to not pass in account infos, but to pass in refs to account data (for those accounts that attempt to deserialize in the helper method) or deserialized data (in the case of the auction and custody token accounts) with relevant pubkeys (since the keys are used for instructions to CPI).

We can discuss this more offline.